packages/manager/apps/pci-public-ip/src/api/hooks/useUser.ts

```diff
@@ -0,0 +1,20 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+import { useQuery } from '@tanstack/react-query';
+
+export interface User {
+  ovhSubsidiary: string;
+}
+export const getUser = async (): Promise<User> => {
+  const { data } = await v6.get(`/me`);
+  return data as User;
+};
+
+export const getUserQuery = () => ({
+  queryKey: ['me'],
+  queryFn: () => getUser(),
+});
+
+export const useUser = () =>
+  useQuery({
+    ...getUserQuery(),
```

Comment:
it was a copy/paste, i don't use this hook anymore, no need to make an extra http call to load the user since it's already in the context, i just happened to discover that and created a hook called useMe that uses the data already fetched by the shell context, planning to move it later to shared project, imma remove the spread here for the other pieces of code that are still using it , i'm still being onboarded to the project and still making those types of discoveries that should be made a long time ago, a lot of http calls are redundant